src/api/api.js

```diff
@@ -0,0 +1,35 @@
+const getUserData = async () => {
+  const res = await fetch("https://bootcamp-api.codeit.kr/api/sample/user");
```

Comment:
```suggestion
const API_BASE_URL = 'https://bootcamp-api.codeit.kr' // 최상단에 선언하기
  const res = await fetch(`${API_BASE_URL }/api/sample/user`);
```

같은 식으로 baseUrl을 미리 선언해주는게 오타 방지 및 재사용에도 도움이 될겁니다 참고 부탁드려요~

Reply:
009eaf98bd9f2f0cd09c1ccf587efa7e843e48f7 수정하였습니다! 오타도 방지되고 재사용하기 편리해서 좋은 것 같습니다!